.github/workflows/conda.yml

```diff
@@ -4,10 +4,6 @@ on:
     branches:
       - main
   pull_request:
-    paths:
-      - setup.py
-      - continuous_integration/recipes/**
-      - .github/workflows/conda.yml
```

Comment:
Would hang onto these

Reply:
They seem to be back https://github.com/dask/distributed/pull/6104/files#diff-72b980456954eec0c566d17ce7746f2c1f7c6b9b98b1ad01395b6f7cb8c4ca0b (I think the merge was a bit weird here...)

Reply:
Ah ok. Yep looks better now. Thanks Martin!